src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
```

Comment:
The `Main` class should not contain any fields. It is meant to only contain the `main` method. Remove the `figureSupplier` field from here and initialize it inside the `main` method.